app/screens/integration_selector/channel_list_row/index.tsx

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {
+    Text,
+    View,
+} from 'react-native';
+
+import CompassIcon from '@components/compass_icon';
+import {makeStyleSheetFromTheme, changeOpacity} from '@utils/theme';
+
+import CustomListRow, {Props as CustomListRowProps} from '../custom_list_row';
+
+type ChannelListRowProps = {
+    id: string;
+    theme: object;
+    channel: Channel;
+    onPress: (item: Channel) => void;
+};
+
+type Props = ChannelListRowProps & CustomListRowProps;
+
+const getStyleFromTheme = makeStyleSheetFromTheme((theme) => {
+    return {
+        titleContainer: {
+            alignItems: 'center',
+            flexDirection: 'row',
+        },
+        displayName: {
+            fontSize: 16,
```

Comment:
If you don't mind I'll start using typography once I get into the design for the V2 since I haven't done any styling changes here and I'd rather focus on the functionality now.